test/e2e/framework/ingress_utils.go

```diff
@@ -1109,9 +1189,20 @@ func (j *IngressTestJig) GetRootCA(secretName string) (rootCA []byte) {
 
 // TryDeleteIngress attempts to delete the ingress resource and logs errors if they occur.
 func (j *IngressTestJig) TryDeleteIngress() {
-	err := j.Client.ExtensionsV1beta1().Ingresses(j.Ingress.Namespace).Delete(j.Ingress.Name, nil)
+	j.TryDeleteGivenIngress(j.Ingress)
```

Comment:
TryDeleteIngress is the only caller for TryDeleteGivenIngress and all it does is call that method.
Do we really need 2 methods?

Reply:
Ah, I originally added `TryDeleteGivenIngress` for the scale test, but ended up implementing its own cleanup function. So you are right it is not needed now...And sorry for making you rebase :)